content/blog/esc-automation-api-pulumi-service-provider-launch/index.md

```diff
@@ -0,0 +1,316 @@
+---
+title: "Introducing Pulumi ESC Support in Automation API and the Pulumi Service Provider"
+allow_long_title: true
+date: 2024-06-05T00:00:00-07:02
+draft: false
+meta_desc: "The new enhancements to Pulumi ESC Editor streamlines the authoring experience of environments for developers"
+meta_image: "meta.png"
+authors:
+  - arun-loganathan
+  - iaroslav-titov
+tags:
+  - esc
+  - secrets
+  - config management
+  - features
+---
+
+We're excited to announce two powerful new capabilities for [Pulumi Environments Secrets and Configurations](/product/esc) (ESC) that supercharge how you manage and control your infrastructure and application secrets and configurations:
+
+- **Pulumi Service Provider Support for ESC Environments**: Manage your environments as code right alongside your infrastructure including fine-grained access control using [Pulumi Service Provider](/registry/packages/pulumiservice/).
+- **Automation API Enhancements for ESC**: Seamlessly integrate ESC Environments into your [Automation API](/automation/)-based Pulumi projects.
+  
+<!--more-->
+
+These updates build upon Pulumi ESC's commitment to simplify and streamline secrets and configuration management providing you with greater control, security, and automation capabilities.
+
+## Manage ESC Environments as Code with the Pulumi Service Provider
+
+You can now manage your Pulumi ESC [Environments](/docs/esc/environments/) directly within the Pulumi Service using the powerful Pulumi Service Provider. This means you can define, version, tag your environments, and even control access using familiar Infrastructure as Code (IaC) practices ensuring consistency and repeatability across your deployments.
+
+Here's a glimpse of what you can achieve:
+
+- **Programmatically create and manage ESC Environments**: Define environments alongside your infrastructure code for a unified workflow.
+- **Add version tags to environments**: Add tags when you are creating the environment.
+- **Enforce granular team permissions**: Control who can [read, open, or modify](/docs/esc/environments/#organization-wide-permissions) your environments ensuring secure collaboration and preventing unauthorized access.
+
+Bringing Pulumi ESC Environments support into the Pulumi Service Provider empowers you to manage your entire infrastructure and application landscape through a unified approach.
+
+Pulumi ESC Automation API capabilities are available for Typescript, Go, Python, C#, Java, and YAML. See our [Environment](registry/packages/pulumiservice/api-docs/environment/) and [Environment Permissions](registry/packages/pulumiservice/api-docs/teamenvironmentpermission/) Pulumi Service Provider docs. Here are examples of creating environments in Typescript, Go, and Python:
```

Comment:
You are talking about PSP in the paragraph, I think Automation API stuff is leaked into here?

Reply:
thanks for spotting. I updated it!